components/Org/OrgHistoryPanel.js

```diff
@@ -0,0 +1,58 @@
+import OpList from '../Op/OpList'
+import PropTypes from 'prop-types'
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import ReduxLoading from '../Loading'
+import { Alert } from 'antd'
+import { orgHistoryTitle, orgHistoryNotFound, orgHistoryError } from './OrgHistoryPanel.messages'
+import reduxApi from '../../lib/redux/reduxApi'
+
+export const OrgHistoryPanel = ({ organisationId }) => {
+  const archivedOpportunities = useSelector(state => state.archivedOpportunities.data)
+
+  const loadingState = useSelector(state => ({
+    sync: state.archivedOpportunities.sync,
+    syncing: state.archivedOpportunities.syncing,
+    loading: state.archivedOpportunities.loading,
+    error: state.archivedOpportunities.error
+  }))
+
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    if (!loadingState.loading) {
+      dispatch(
+        reduxApi.actions.archivedOpportunities.get({ q: JSON.stringify({ offerOrg: organisationId }) })
+      )
+    }
+  }, [organisationId])
+
+  let content = ''
+
+  if (loadingState.loading || (!loadingState.sync && !loadingState.error)) {
+    content = <ReduxLoading />
+  } else if (loadingState.error) {
```

Comment:
Why bypass and then reproduce the behaviour of <Loading> ?

Reply:
I didn't really think about it at the time but trying to use the `ReduxLoading` component to handle the error now the main problem I'm seeing is that when the `OrgHistoryPanel` component first renders the initial state of the Redux store for archivedOpportunities is:

    { sync: false, syncing: false, loading: false, error: undefined, data: [] }

So the `ReduxLoading` component displays the "not synced" message for a split second before the Redux store updates the `loading` flag to `true`.

In the app this appears as a flash of a warning message, followed by the loading animation, and finally the records show up, e.g.

![vly-org-history-warning-flash](https://user-images.githubusercontent.com/3037783/76560140-66133c80-6505-11ea-8c37-c4d635f0aaa7.gif)

I'd like to avoid that flash of a warning message - especially in this case where it's always going to happen due to the initial state of the redux store. I appreciate the message only shows in development environments but I don't think it should be shown at all when things are working as intended.